docs/core/deploying/deploy-with-vs.md

```diff
@@ -104,7 +104,7 @@ Once you've finished debugging, you can publish your self-contained deployment:
 
 <!-- markdownlint-disable MD025 -->
 
-# [Visual Studio 15.6 and earlier](#tab/vs156)
+# [Visual Studio version 15.6 and earlier](#tab/vs156)
```

Comment:
we'd probably need to get this tab name approved first

Reply:
Okay, I'll just revert this then.